packages/cloudvision-connector/src/Connector.ts

```diff
@@ -190,7 +202,7 @@ export default class Connector extends Wrpc {
   /**
    * Returns a list of apps (datasets with type == 'app').
    */
-  public getApps(callback: NotifCallback): string | null {
+  public getApps(callback: NotifCallback): SubscriptionIdentifier | null {
```

Comment:
Can you rename this type to make it's name more representative of what it is now? something like `CallIdentifier`